op/framework.py

```diff
@@ -156,9 +156,7 @@ def emit(self, *args, **kwargs):
         The current storage state is committed before and after each observer is notified.
         """
         framework = self.emitter.framework
-        # TODO This needs to be persisted.
-        framework._event_count += 1
-        key = str(framework._event_count)
+        key = str(framework.event_count)
```

Comment:
This shouldn't be public. There's no reason for anyone outside of the framework itself to read or write it.